stablehlo/dialect/StablehloOps.cpp

```diff
@@ -594,10 +599,66 @@ ParseResult parsePrecisionConfig(OpAsmParser& parser, mlir::ArrayAttr& attr) {
           })))
     return failure();
 
-  attr = mlir::ArrayAttr::get(parser.getContext(), attrs);
+  precision = mlir::ArrayAttr::get(parser.getContext(), attrs);
+  return success();
+}
+
+void printPrecisionConfigAndAlgorithm(OpAsmPrinter& p, Operation* op,
```

Comment:
This function isn't used? If this is helpful, maybe this can be used in `parsePrecisionConfigAndAlgorithm` on error.

Reply:
The print and parse are both used in generated code by `custom<PrecisionConfigAndAlgorithm>`